lib/src/metta/runner/stdlib.rs

```diff
@@ -146,6 +146,101 @@ fn strip_quotes(src: &str) -> &str {
     src
 }
 
+/// Provides a way to access [Metta::load_module_at_path] from within MeTTa code
+#[derive(Clone, Debug)]
+pub struct RegisterModuleOp {
+    metta: Metta
+}
+
+impl PartialEq for RegisterModuleOp {
+    fn eq(&self, _other: &Self) -> bool { true }
+}
+
+impl RegisterModuleOp {
+    pub fn new(metta: Metta) -> Self {
+        Self{ metta }
+    }
+}
+
+impl Display for RegisterModuleOp {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "register-module!")
+    }
+}
+
+impl Grounded for RegisterModuleOp {
+    fn type_(&self) -> Atom {
+        Atom::expr([ARROW_SYMBOL, ATOM_TYPE_ATOM, UNIT_TYPE()])
+    }
+
+    fn execute(&self, args: &[Atom]) -> Result<Vec<Atom>, ExecError> {
+        let arg_error = "load-module! expects a file system path; use quotes if needed";
```

Comment:
A minor issue with `load-module!` instead of `register-module`.

Reply:
In addition, my proposal (although not explicitly stated) was to have `register-module` without `!`. While this function does have side effects, they are indirect (no immediate effect on any space or external file). I'm ok with keeping `!` as well, but wish to know what do you think about removing `!`?

Reply:
> A minor issue with load-module! instead of register-module.

Oops.  Good catch.  Fixed in 36de03d1e3e308f180b6cc0c98db90e6aa1fc811

> So the '!' at the end of an operation indicates it's not a pure function?  I didn't make that connection until you mentioned it.  But it makes sense.

`register-module!` definitely has side effects that are at least as meaningful as the side-effects of other operations like `println!`, so I think it makes sense to keep the '!'